### PR TITLE
Add Czech language

### PR DESCRIPTION
**Pull request by Fjuro**

### CODE REVIEW

File: app/src/main/res/values-cs/strings.xml [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    <string name="message_backup_create_success">Záloha úspěšně vytvořena</string>
+    <string name="message_backup_create_failed">Nepodařilo se vytvořit zálohu</string>
+    <string name="message_restore_failed">Nepodařilo se obnovit zálohu</string>
+
+    <!-- Notification -->
+    <string name="channel_name_playback">Hudební přehrávač</string>
+    <string name="channel_name_download">Stáhnout</string>
+
+    <!-- Noun -->
+    <plurals name="song_count">
```
Comment (z-huang) [inline app/src/main/res/values-cs/strings.xml:180]: the following quantities should also be defined: few

**Review by z-huang — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-cs/strings.xml (hunk 1 above)
Comment (Fjuro) [reply app/src/main/res/values-cs/strings.xml:180]: Done

Comment (z-huang) [reply app/src/main/res/values-cs/strings.xml:180]: The build says "the following quantity should also be defined: many".

Comment (Fjuro) [reply app/src/main/res/values-cs/strings.xml:180]: Okay, done, too.

Comment (Fjuro) [reply app/src/main/res/values-cs/strings.xml:180]: Oops, didn't see those as well, sorry, I'll check the whole file

**Review by z-huang — APPROVED** [review]

**Comment by rajibulislam1928** [comment]
Please add sleeper time

**Comment by z-huang** [comment]
Added in `compose` branch.

**Pull request merged**